<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStationManager</class>
 <widget class="QMainWindow" name="CStationManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>1891</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_menue"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1921</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>2140</width>
      <height>982</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="MenuBtn">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Pictures/隐藏.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>980</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>980</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 40, 77);
color: rgb(255, 255, 255);
font: 11pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>980</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>980</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="page">
        <property name="minimumSize">
         <size>
          <width>1900</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1900</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QScrollArea" name="scrollArea">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1840</width>
           <height>1021</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>1700</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1840</width>
           <height>1080</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>false</bool>
         </property>
         <widget class="CStationDraw" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>70000</width>
            <height>1080</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>70000</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70000</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="minimumSize">
         <size>
          <width>1920</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1920</width>
           <height>1011</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>1920</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1920</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="jsj_UserManagerdlg" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1920</width>
            <height>1050</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>1920</width>
            <height>1050</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1920</width>
            <height>1050</height>
           </size>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="minimumSize">
         <size>
          <width>1920</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>16777215</width>
           <height>16777215</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="JSJ_ddmldlg" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1920</width>
            <height>1500</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>1920</width>
            <height>1500</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1920</width>
            <height>1500</height>
           </size>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CStationDraw</class>
   <extends>QWidget</extends>
   <header>stationdraw.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jsj_UserManagerdlg</class>
   <extends>QWidget</extends>
   <header>jsj_usermanagerdlg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JSJ_ddmldlg</class>
   <extends>QWidget</extends>
   <header>jsj_ddmldlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>scrollArea_3</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
